versions.yaml

```diff
@@ -289,7 +289,7 @@ languages:
       description: |
         'newest-version' is the latest version known to work when
         building Kata
-      newest-version: "1.44.1"
+      newest-version: "1.46.0"
```

Comment:
I think here can let it remain to 1.44.1, and only let `rust-init` install version 1.44.1  but not install the latest one first.

Reply:
@liubin ok, i revert rust version to 1.44.1 and recommit. :)